package.json

```diff
@@ -45,7 +45,8 @@
     "ultrahook": "export $(grep ^ULTRAHOOK_API_KEY .env | tr -d \"'\") && ultrahook -k $ULTRAHOOK_API_KEY dev 3000",
     "precommit": "lerna run --stream precommit",
     "postcheckout": "node scripts/generateGraphQLArtifacts.js &>/dev/null &",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "test:server": "yarn workspace parabol-server test"
```

Comment:
+1 It would be nice if you could adapt '.circleci/config.yml' to call this now, so we're sure it's tested

Reply:
✅